### ISSUE TITLE
Could you please share a sample of your training data?

### ISSUE THREAD

**Issue by JacCubk** [vietthanh2710/AC_MambaSeg#3]
Hello, could you please share a sample of your training data? I'm trying to run your code but I'm unsure how to prepare the dataset. It seems you stored 'image' and 'mask' in an npz file, but I keep encountering errors when I try to do the same.

**Comment by vietthanh2710** [comment]
Thats right, the data is basically saved as an npz file with 2 keys that are "image" and "mask". But you can write your own dataloader preparation. 
Here a sample npz file. [https://drive.google.com/file/d/1-4YteRbgH_iD6IW4w928BVqbYS6vC5_R/view?usp=sharing](url)

**Comment by JacCubk** [comment]
Thank you for providing the data sample, it means a lot to me. I have found that there might be an issue with my environment, as the program goes into an infinite loop regardless of whether I use the data you provided or my own data. This is all too frustrating, and I am eager to know about the environment you are using, at your convenience. I would greatly appreciate your generosity.

**Comment by liyuqiao-888** [comment]
您好，可以分享一下这个npz文件吗？

**Comment by liyuqiao-888** [comment]
Hello, could you please share a sample of your training data? I'm trying to run your code but I'm unsure how to prepare the dataset. It seems you stored 'image' and 'mask' in an npz file, but I keep encountering errors when I try to do the same.

**Comment by liyuqiao-888** [comment]
> 您好，您能否分享一下您的训练数据样本？我正在尝试运行您的代码，但我不确定如何准备数据集。您似乎将“image”和“mask”存储在 npz 文件中，但是当我尝试执行相同操作时，我不断遇到错误。

**Comment by ytuv** [comment]
> 感谢您提供数据样本，这对我来说意义重大。我发现我的环境可能存在问题，因为无论我使用的是您提供的数据还是我自己的数据，程序都会进入无限循环。这太令人沮丧了，我很想知道您在方便的时候正在使用的环境。我非常感谢您的慷慨解囊。

您好，您能分享一下提供的数据嘛？不好意思想请问一下您复现出来的结果和原文一致嘛？为什么我复现出来的指标值都很低？不知道是不是数据集出现的问题还是哪里出现的问题？希望得到您的回复，感谢！（Hello, can you share the data provided? Sorry, I would like to ask if the results you reproduced are the same as the original text? Why are the index values I reproduced very low? I don't know if there is a problem with the dataset or somewhere? Hope to get your reply, thank you.）

**Comment by vietthanh2710** [comment]
You can refer to this sample of ISIC2018 dataset [sample](https://drive.google.com/file/d/1v8NyN-CKe-XUQhfyRLNxWpOnbSP8JbNl/view?usp=sharing)

**Comment by ytuv** [comment]
> 您可以参考此 ISIC2018 数据集示例 [https://drive.google.com/file/d/1v8NyN-CKe-XUQhfyRLNxWpOnbSP8JbNl/view?usp=sharing](url)

不好意思，该链接打开之后没有文件显示
![Uploading 屏幕截图 2024-12-11 160537.png…]()

**Comment by vietthanh2710** [comment]
@ytuv I just updated the link, now you can acccess to the data sample.

**Comment by ytuv** [comment]
> @ytuv I just updated the link, now you can acccess to the data sample.

Could  you please  share the  isic2018 dataset to my email :Wangytuv@163.com ，I cant  open  the  link  you  sent .I really  appreciate    your   help .

**Comment by ytuv** [comment]
> @ytuv I just updated the link, now you can acccess to the data sample.

Sorry ,I can  not open  the  link you sent .Could you  please  share  me  the  ISIC2018 datasets  to my  email :wangytuv@163.com ,I really  appreciate  for  your help !
